mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/impl/MarcFieldNonRepeatable1xxFieldRuleValidator.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.rspec.validation.validator.marc.impl;
+
+import java.util.List;
+import org.folio.rspec.domain.dto.DefinitionType;
+import org.folio.rspec.domain.dto.SeverityType;
+import org.folio.rspec.domain.dto.SpecificationFieldDto;
+import org.folio.rspec.domain.dto.ValidationError;
+import org.folio.rspec.i18n.TranslationProvider;
+import org.folio.rspec.validation.validator.SpecificationRuleCode;
+import org.folio.rspec.validation.validator.SpecificationRuleValidator;
+import org.folio.rspec.validation.validator.marc.model.MarcField;
+import org.folio.rspec.validation.validator.marc.model.MarcRuleCode;
+import org.folio.rspec.validation.validator.marc.utils.TagsMatcher;
+
+class MarcFieldNonRepeatable1xxFieldRuleValidator
+  implements SpecificationRuleValidator<MarcField, SpecificationFieldDto> {
+
+  private final TranslationProvider translationProvider;
+
+  MarcFieldNonRepeatable1xxFieldRuleValidator(TranslationProvider translationProvider) {
+    this.translationProvider = translationProvider;
+  }
+
+  @Override
+  public List<ValidationError> validate(MarcField marcField, SpecificationFieldDto field) {
+    if (TagsMatcher.matches1xx(marcField.tag())
+      && Boolean.FALSE.equals(field.getRepeatable()) && marcField.reference().getTagIndex() > 0) {
+      return List.of(buildError(marcField, field));
+    }
+    return List.of();
+  }
```

Comment:
This rule must validate that record has only one field from 1XX group. For example:
if record has one 100 field and one 199 field then it will fail.

Reply:
Fixed